src/app/producer-shelf/services/hardcoded-producer-keyboard-binding.service.ts

```diff
@@ -0,0 +1,145 @@
+import { KeyboardBindingService } from '../abstractions/keyboard-binding.service'
+import { BehaviorSubject, fromEvent, Subject, Subscription } from 'rxjs'
+import { KeyBinding } from '../models/key-binding'
+import { KeyboardBindingMatcher } from './keyboard-binding.matcher'
+import { HostListener, Injectable } from '@angular/core'
+import { CameraKeyBindingFactory } from '../factories/camera-key-binding.factory'
+import { RundownService } from '../../core/abstractions/rundown.service'
+
+@Injectable()
+export class HardcodedProducerKeyboardBindingService implements KeyboardBindingService {
```

Comment:
I assume this class is for the demo purposes of the keyboard?

If yes, I'm not going to spend much time reveiwing it